Manual.md

```diff
@@ -76,17 +76,19 @@ packages for XBPS, the `Void Linux` native packaging system.
 <a id="Introduction"></a>
 ## Introduction
 
-The `void-packages` repository contains all `source` packages that are the
-recipes to download, compile and build binary packages for `Void`.
-Those `source` package files are called `templates`.
+The `void-packages` repository contains all the
+recipes to download, compile and build binary packages for Void Linux.
+These `source` package files are called `templates`.
 
-The `template files` are `GNU bash` shell scripts that must define some required/optional
```

Comment:
Simply "shell scripts that define variables and functions that are processed by `xbps-src`" maybe?

Reply:
I still wanted to mention bash, let me know if it's okay this way.

Reply:
Both forms of mentioning bash are fine. Just wanted clean up `required or optional` quirk.